frontend/public/locales/en/public.json

```diff
@@ -824,14 +824,12 @@
   "These versions are supported, but include known issues. Review the known issues before updating.": "These versions are supported, but include known issues. Review the known issues before updating.",
   "Updating this cluster to {{desiredVersion}} is supported, but includes known issues.  Review the known issues before updating.": "Updating this cluster to {{desiredVersion}} is supported, but includes known issues.  Review the known issues before updating.",
   "Update options": "Update options",
-  "Full cluster update allows you to update all your Nodes, but takes longer. Partial cluster update allows you to pause worker and custom pool Nodes to accommodate your maintenance schedule, but you'll need to resume the non-control plane Node updates within 60 days to avoid failure.": "Full cluster update allows you to update all your Nodes, but takes longer. Partial cluster update allows you to pause worker and custom pool Nodes to accommodate your maintenance schedule, but you'll need to resume the non-control plane Node updates within 60 days to avoid failure.",
+  "Full cluster update allows you to update all your Nodes, but takes longer. Partial cluster update allows you to pause worker and custom pool Nodes to accommodate your maintenance schedule.": "Full cluster update allows you to update all your Nodes, but takes longer. Partial cluster update allows you to pause worker and custom pool Nodes to accommodate your maintenance schedule.",
```

Comment:
just wondering if we should also be backporting the other i18n translations of this string as a part of this bugfix, or if it's okay for this string to always be displayed in English